core/componentConfig.go

```diff
@@ -66,16 +66,20 @@ func (cc *ComponentConfig) MergeConfigFile(path string, environment string) (err
 func (cc *ComponentConfig) Load(environment string) (err error) {
 	err = cc.UnmarshalYAMLConfig(environment)
 
-	// fall back to looking for JSON if loading YAML fails.
-	if err != nil {
-		err = cc.UnmarshalJSONConfig(environment)
+	// If success or loading or parsing the file failed for reasons other than it didn't exist, return.
+	if err == nil || !os.IsNotExist(err) {
```

Comment:
can you change to

```go
if err = err = cc.UnmarshalYAMLConfig(environment); err == nil || !os.IsNotExist(err) {
    return err
}
```

like to keep our style for immediate error returns consistent

Reply:
oh great catch - yes - will do